test/roundtrip/fold-try-delegate.txt

```diff
@@ -1,28 +1,31 @@
 ;;; TOOL: run-roundtrip
 ;;; ARGS: --stdout --fold-exprs --enable-exceptions --debug-names
 (module
-  (func (result i32)
-    try (result i32)
+  (func
+    try
       try
         nop
-      delegate 1
-      i32.const 7
+      delegate 0
```

Comment:
`WatWriter::ExprVisitorDelegate::OnDelegateExpr` doesn't seem to be used to print folded wat format, so this test isn't really affected. (This test succeeds even now.)

Reply:
Thanks I've just removed the changes to this test in the latest push and kept only the necessary one.